src/style.css

```diff
@@ -1,3 +1,82 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  margin: 0;
+}
+
+.img__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: inline-block;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  font-size: 0;
```

Comment:
why do we need this

Reply:
fixed this and made display: flex instead of inline-block